test/distrib/bazel/cpp/tools/bazel

```diff
@@ -0,0 +1 @@
+../../../../../tools/bazel
```

Comment:
add a comment that explains the idea behind this redirection (I understand, but not everyone will).

Reply:
Ah ok, thats fair then.